### DISCUSSION TITLE
Task 1 Test

### DISCUSSION THREAD

**Discussion by cuteyyt** [FETS-AI/Challenge#69] [Q&A] [answered]
Dear Organizers:
Thank you very much for your hard work in organizing this challenge.

As I have noticed that the save checkpoint function is very time-consuming, I wonder whether participants are tested with saving checkpoint in the testing phase or not?

**Comment by sarthakpati** [comment, marked as answer]
> whether participants are tested with saving checkpoint in the testing phase or not?

Hi,

Thanks for your interest in the FeTS Challenge!

We will keep checkpointing enabled for all participants and the testing will happen using the same configuration for all participants, ensuring fair play. 😄 

Cheers,
Sarthak

**Reply by cuteyyt** [reply]
Thanks a lot, and I'm very sorry for not replying in time after checking the emails.
